mock/py/mockbuild/buildroot.py

```diff
@@ -60,6 +60,9 @@ def __init__(self, config, uid_manager, state, plugins, bootstrap_buildroot=None
         self.env.update(proxy_env)
         os.environ.update(proxy_env)
 
+        self.use_bootstrap_image = self.config['use_bootstrap_image']
+        self.bootstrap_image = self.config['bootstrap_image']
```

Comment:
This needs to have defaults in utils.py and documented (commented out default) in `site-defaults.cfg`

Reply:
> > Problem 1: package dnf-yum-4.2.7-2.fc30.noarch conflicts with yum < 3.4.3-505 provided by yum-3.4.3-163.el7.centos.noarch
> 
> On the other hand using fedora:latest to install el7 is... I really do not want to support this. And therefore not even test it. We should use the image of the target system. I.e. centos:7 for epel-7* It can be even part of the default config as setting `bootstrap_image` will make no harm/change untill we set `use_bootstrap_image`.

Do you want me set 'bootstrap_image' in all config files?
Finding and testing the images might be a bit challenging and definitely will take time.

Reply:
a) it can be done in separate PR b) I and @frostyx can help with the testing